### ISSUE TITLE
Userspace WireGuard-go in Gateway Docker Image

### ISSUE THREAD

**Issue by drag0n141** [DefGuard/gateway#136]
Hi,

i want to use `DEFGUARD_USERSPACE=true` in the docker container. But it does not work, the wireguard-go package is missing. I am using the latest container.

```
[2024-11-15T19:18:45Z ERROR defguard_wireguard_rs::wgapi_userspace] Failed to create userspace API. wireguard-go executable not found in PATH. Error: No such file or directory (os error 2)
Error: WireguardError(ExecutableNotFound("wireguard-go"))
```

**Comment by teon** [comment]
@drag0n141 yes - official docker images are ment for Linux distributions which have WireGuard in Kernel, thus we don't include the fireguard-go in the docker image.
What would be the case for such a use case?

**Comment by drag0n141** [comment]
I am want to run the gateway in my kubernetes cluster. For that I need the userspace mode.
